README.md

```diff
@@ -10,50 +10,281 @@
+
+Examples:
+* `edit student m/CS2103 i/A0123456A n/John Doe`:
+  * Edit the name of the student with id A0123456A in module CS2103 to John Doe
+* `edit student m/CS2100 i/A0123457A t/@amylee e/amy@u.nus.edu`:
+  * Edit the tele handle and email of the student with id A0123457A in module CS2100 to @amylee and amy@u.nus.edu respectively
+
+### Delete a student from a specific module: `delete student`
+
+Delete the specified student from the specified module.
+
+Format: `delete student m/<module name> i/<student id>`
+
+Examples:
+* `delete student m/CS2103 i/A1234567A`:
+  * Delete student with id A1234567A from module CS2103’s list of students
+* `delete student m/CS2100 i/A7654321A`:
+  * Delete student with id A7654321A from module CS2100’s list of students
+
+### Add a new task to a module: `add task`
+
+Add a new task to the specified module.<br>
+Task id must begin with a capital letter 'T', followed by numbers only. The numbers must not start with 0.<br>
+Task name must only contain alphanumeric characters and spaces, and it must not be blank.<br>
+Task deadline must contain either date only/time only/date and time.<br>
+Date must be in the form "YYYY-MM-DD".<br>
+Time must be in the form "hh:mm" in 24-hrs format, and will appear in TAB in 12-hrs format.
+
+Format: `add task m/<module name> ti/<task id> a/<task name> d/<task deadline>`
+
+Examples:
+* `add task m/CS2103 ti/T1 a/assignment1 d/2021-10-12`:
+  * Add the task assignment1 with deadline 'Tue, Oct 12 2021' to module CS2103
+* `add task m/CS2100 ti/T1 a/lab1 d/2021-10-13 16:00 `:
+  * Add the task lab1 with deadline 'Wed, Oct 13 2021 04:00 pm' to module CS2100
+
+### Edit a task's information: `edit task`
+
+Edit a task's information.<br>
+The identity parameters (module name/task id) must be provided.<br>
+At least one optional editable parameter (name/deadline) must be provided, and only provided editable parameters will be changed.<br>
+If the editable parameters contain the same information as is already present, the edit command will still be executed successfully.<br>
+Task name must only contain alphanumeric characters and spaces, and it must not be blank.<br>
+Task deadline must contain either date only/time only/date and time.<br>
+Date must be in the form "YYYY-MM-DD".<br>
+Time must be in the form "hh:mm" in 24-hrs format, and will appear in TAB in 12-hrs format.
+
+Format: `edit task m/<module name> ti/<task id> [a/<edited task name>] [d/<edited task deadline>]`
+
+Examples:
+* `edit task m/CS2103 ti/T10 a/Final exam d/2021-11-23`:
+  * Edit the name and deadline of the task with task id 'T10' to 'Final exam' and 'Tue, Nov 23 2021' respectively
+* `edit task m/CS2103 ti/T10 d/2021-11-23 17:00`:
+  * Edit only the deadline of the task with task id 'T10' to 'Tue, Nov 23 2021 5:00 pm'
+
+### Delete a task from a module: `delete task`
+
+Delete the specified task from the specified module.<br>
```

Comment:
should we mention something like `by providing the task ID and module name`? this is to explain the `specified` used here.

Reply:
The same goes for all the sentences below that use the word `specified`.

Reply:
Agreed, will specify